pallets/grant/src/tests.rs

```diff
@@ -21,15 +22,28 @@ fn next_block(n: u64) {
 	Grant::on_initialize(n);
 }
 
+fn treasury_account() -> AccountId {
+	<Test as Config>::TreasuryAccount::get()
+}
+
+fn fund_treasury(u: u64) {
+	let _ = Balances::mutate_account(&treasury_account() ,|amount| {
+		amount.free += u
+	});	
+}
+
+fn grant_amount() -> Balance {
+	<Test as Config>::GrantAmount::get()
+}
 
 // <<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<< TESTS >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
 
 #[test]
 fn accounts_can_request_a_grant() {
 	new_test_ext().execute_with(|| {
 
-		// Ensure we can request grants
-		assert_ok!(Grant::request_grant(Origin::signed(1), 7 ));
+		// Ensure we can request grants with empty balance.
+		assert_ok!(Grant::request_grant(Origin::signed(12345)));
```

Comment:
We are using `use once_cell::sync::Lazy;` to preload some accounts. You can check the mock for dao for example. You can also load it as dev dependency to grant pallet and use *ALICE reference instead of `12345`

Reply:
https://github.com/UniversalDot/universal-dot-node/pull/107/commits/803c135f756184030d5f8038ecde54166bcf0262